app-admin/vaultwarden/files/vaultwarden-envfile-1.33.2.patch

```diff
@@ -0,0 +1,62 @@
+--- a/.env.template
```

Comment:
I think we need a little blurb here to explain why we are modifying this template